model/trace/messaging.yaml

```diff
@@ -205,25 +205,106 @@ groups:
+        tag: tech-specific
       - ref: messaging.servicebus.disposition_status
         requirement_level:
           conditionally_required: if and only if `messaging.operation` is `settle`.
+        tag: tech-specific
+      - ref: server.address
+        brief: >
+          The domain name of the Service Bus namespace.
+        examples: ["contoso.servicebus.windows.net"]
+        note: >
+          <!-- TODO: overriding the base note, workaround for https://github.com/open-telemetry/build-tools/issues/299 -->
+        tag: tech-specific
+      - ref: error.type
+        note: >
+          The error type SHOULD match the AMQP error code (when applicable) or a fully qualified type name
+          of the exception that caused the error.
+        examples: ["amqp:not-found", "com.microsoft:server-busy", "java.net.UnknownHostException"]
+        tag: tech-specific
+      - ref: messaging.destination.name
+        brief: >
+          The name of the Service Bus queue or topic the message is sent to or received from.
+        note: >
+          <!-- TODO: overriding the base note, workaround for https://github.com/open-telemetry/build-tools/issues/299 -->
+        tag: tech-specific
+        requirement_level:
+          conditionally_required: If applicable.
+      - ref: messaging.batch.message_count
+        tag: tech-specific
+      - ref: messaging.operation.type
+        tag: tech-specific
+      - ref: messaging.operation.name
+        examples: ["send", "process", "get_partition_properties"]
+        note: |
+          The operation name SHOULD match one of the following values (the normalized name of the client library method in snake_case):
+
+          - sender operations: `send`, `schedule`, `cancel_scheduled`
+          - transaction operations: `create_transaction`, `commit_transaction`, `rollback_transaction`
+          - receiver operation: `receive`, `peek`, `receive_deferred`, `renew_message_lock`
+          - settlement operations: `abandon`, `complete`, `defer`, `dead_letter`
+          - session operation: `accept_session`, `get_session_state`, `set_session_state`, `renew_session_lock`
+        tag: tech-specific
+
   - id: messaging.eventhubs
     type: attribute_group
     extends: messaging
     brief: >
       Attributes for Azure Event Hubs
     attributes:
-      - ref: messaging.destination.partition.id
-        brief: >
-          String representation of the partition id messages are sent to or received from, unique within the Event Hub.
-        requirement_level:
-          conditionally_required: If available.
       - ref: messaging.eventhubs.message.enqueued_time
+        tag: tech-specific
       - ref: messaging.eventhubs.consumer.group
         requirement_level:
           conditionally_required: If not default ("$Default").
+        tag: tech-specific
+      - ref: server.address
+        brief: >
+          The domain name of the Event Hubs namespace.
+        examples: ["contoso.servicebus.windows.net"]
```

Comment:
```suggestion
        examples: ["contoso.eventhubs.windows.net"]
```

Reply:
servicebus and eventhubs resources are both in the `servicebus.windows.net` domain :)